### PR TITLE
add voter entity type

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2026
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/327

### CODE REVIEW

File: Backoffice/Security/Authorization/Voter/AbstractEditorialVoter.php [hunk 1]
```diff
@@ -61,6 +62,10 @@ protected function voteOnAttribute($attribute, $subject, TokenInterface $token)
             return true;
         }
 
+        if (ContributionActionInterface::CREATE === $attribute) {
```
Comment (open-orchestra-admin) [inline Backoffice/Security/Authorization/Voter/AbstractEditorialVoter.php]: @alavieille : why not merge the READ & READ conditions with a II operator ?

File: Backoffice/Security/Authorization/Voter/AbstractVoter.php [hunk 2]
```diff
@@ -46,7 +37,24 @@ protected function supports($attribute, $subject)
...
+     */
+    abstract protected function supportSubject($subject);
+
+    /**
+     * @param mixed $subject
+     * @param array $classes
+     *
+     * @return bool
+     */
+    protected function supportedClasses($subject, array $classes) {
```
Comment (ngilain) [inline Backoffice/Security/Authorization/Voter/AbstractVoter.php]: this should contains a verb like ``isSupportedClass``

**Comment by alavieille** [comment]
@ngilain  pr updatd

**Comment by ngilain** [comment]
:+1:

**Pull request merged**